nautobot_capacity_metrics/metrics.py

```diff
@@ -120,6 +123,31 @@ def metric_models(params):
     yield gauge
 
 
+def metric_versions():
+    """Return django, Nautobot, Python and app versions in Prometheus Metric format.
```

Comment:
Any security concerns around exposing this information? I can't recall whether the app-metrics API endpoint requires authentication or not.

Reply:
Good catch, it is in fact not authenticated.
- Should it be?
- If not, should I create a separate endpoint for sensitive metrics?

Reply:
After some internal discussion I would says that the possibility of toggling this data off and having it toggled off by default should suffice for now.